src/Microsoft.ML.Data/TrainContext.cs

```diff
@@ -346,16 +346,17 @@ internal ClusteringTrainers(ClusteringContext ctx)
         /// <param name="estimator">The estimator to fit.</param>
         /// <param name="numFolds">Number of cross-validation folds.</param>
         /// <param name="labelColumn">Optional label column for evaluation (clustering tasks may not always have a label).</param>
+        /// <param name="featuresColumn">OPtional features column for evaluation (needed for calculating Davies-Bouldin Index)</param>
         /// <param name="stratificationColumn">Optional stratification column.</param>
         /// <remarks>If two examples share the same value of the <paramref name="stratificationColumn"/> (if provided),
         /// they are guaranteed to appear in the same subset (train or test). Use this to make sure there is no label leakage from
         /// train to the test set.</remarks>
```

Comment:
makes more sense to me to have this comment on the strat col param.

Reply:
Why? Stratification column is an argument to `CrossValidateTrain`, not `ClusteringEvaluator.Evaluate`. It does not affect the calculation of BDI.